lib/three_by_three_ttt_player_factory.rb

```diff
@@ -1,20 +1,23 @@
 require_relative '../lib/human_player.rb'
 require_relative '../lib/three_by_three_ttt_computer_player.rb'
+require_relative '../lib/user_console_input_stream.rb'
 
 class ThreeByThreeTTTPlayerFactory
 
-  attr_accessor :number_of_players, :number_of_human_players, :players, :human_player_model, :computer_player_model
+  attr_accessor :number_of_players, :number_of_human_players, :players, :human_player_model, :user_input_stream_framework, :computer_player_model
 
   def initialize(
     number_of_players: 2, 
     number_of_human_players: 1, 
     human_player_model: HumanPlayer,
+    user_input_stream_framework: UserConsoleInputStream,
```

Comment:
1) I like to think about "is a" and "has a" relationships when I think about classes and objects. In this case, I think `ConsoleInputStream` "is an" `InputStream`, since it provides an input stream interface. I think it would be OK in this case to just call this parameter `input_stream`. 

2) Naming critique: `user` seems superfluous in the name of this class. In fact, it might even be misleading—input on stdin usually comes from users typing in a terminal, but can also come from other processes.

Reply:
Just name input_stream as key word

Reply:
Changed names.  Thanks.